torch/_inductor/wrapper_benchmark.py

```diff
@@ -341,26 +350,53 @@ def ncu_analyzer(
         "all",
         "--replay-mode",
         "kernel",
-        "--kernel-name-base",
-        "function",
         "--print-units",
         "base",
-        "--set",
-        "full",
         "--import-source",
         "yes",
         "--force-overwrite",
         "--export",
-        ncu_output,
+        ncu_rep_output,
+    ]
+
+    # Add kernel regex filter if provided
+    if kernel_regex:
+        ncu_cmd.extend(["--kernel-name", f"regex:{kernel_regex}"])
+    else:
+        ncu_cmd.extend(["--kernel-name-base", "function"])
```

Comment:
kernel-name-base is not conflict with kernel-name. they are different things. for example, a vector_add kernel has three names:
function: vector_add
demangled: vector_add(float*, float*, float*)
mangled: _Z10vector_addPfffS_ffS_
`kernel-name-base` decides which name you are going to match with your kernel_regex.

Reply:
Ah thanks for the catch! Fixed it and also removed the csv dump feature.